>From fda3706eb74ba5ad874853969f3df3e372739c8d Mon Sep 17 00:00:00 2001
From: "Xiang, Haihao" <haihao.xiang@intel.com>
Date: Fri, 21 Jun 2013 12:55:30 +0800
Subject: [PATCH] VAEncH264VUIBufferType and VAEncH264SEIBufferType are
 dropped from VA API

The driver doesn't use them indeed

Signed-off-by: Xiang, Haihao <haihao.xiang@intel.com>
---
 src/vdpau_dump.c |    2 --
 1 file changed, 2 deletions(-)

diff --git a/src/vdpau_dump.c b/src/vdpau_dump.c
index 899888b..610e7cd 100644
--- a/src/vdpau_dump.c
+++ b/src/vdpau_dump.c
@@ -59,8 +59,6 @@ const char *string_of_VABufferType(VABufferType type)
         _(VAEncSequenceParameterBufferType);
         _(VAEncPictureParameterBufferType);
         _(VAEncSliceParameterBufferType);
-        _(VAEncH264VUIBufferType);
-        _(VAEncH264SEIBufferType);
 #endif
 #if VA_CHECK_VERSION(0,31,1)
         _(VAQMatrixBufferType);
-- 
1.7.9.5

